cmd/podman/registry/remote.go

```diff
@@ -31,11 +31,11 @@ func IsRemote() bool {
 		fs.SetInterspersed(false)
 		fs.BoolVarP(&remoteFromCLI.Value, "remote", "r", remote, "")
 		connectionFlagName := "connection"
-		ignoredConnection := ""
-		fs.StringVarP(&ignoredConnection, connectionFlagName, "c", "", "")
+		fs.StringP(connectionFlagName, "c", "", "")
+		hostFlagName := "host"
+		fs.StringP(hostFlagName, "H", "", "")
```

Comment:
I'm not a fan of having an upper cased single character option when most (all?) of our others are lower case.  But Docker compatibility, ugh.

Reply:
I assume that -h had already been taken by --help.